docker/docker-compose-l2-explorer.yml

```diff
@@ -79,7 +79,7 @@ services:
       FRONT_PROXY_PASS: http://frontend:3000
       STATS_PROXY_PASS: http://stats:8050/
     ports:
-      - 80:80
+      - 8088:80
```

Comment:
this is needed b/c otherwise `sudo certbot certonly --nginx` will fail with

![telegram-cloud-photo-size-1-5044123014374993588-y](https://github.com/user-attachments/assets/85c575cb-88d9-4231-8dae-557f5590ea92)